ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/utils/useEditorSearch.js

```diff
@@ -1,115 +0,0 @@
-'use strict';
-Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Please remove this file.

Reply:
@laurent22 It looks like the file was accidently added in fd4e5d7, my pull request here will remove it.

Reply:
Oh right, sorry for some reason I thought it was added rather than removed.